app/src/main/java/com/yacgroup/yacguide/RouteActivity.kt

```diff
@@ -62,30 +65,35 @@ class RouteActivity : TableActivityWithOptionsMenu() {
             ClimbingObjectLevel.eCountry to RouteGetter(
```

Comment:
I'm gonna move this huge map into `RouteGetter` in a separate PR (as well as for `RockActivity`) to improve readability.